api/src/opentrons/protocol_runner/json_command_translator.py

```diff
@@ -25,8 +26,12 @@ def _translate_labware_command(
     # v6 data model supports all commands and therefor most props are optional.
     # load labware command must contain labware_id and definition_id.
     assert labware_id is not None
-    definition_id = protocol.labware[labware_id].definitionId
-    assert definition_id is not None
+    try:
+        definition_id = protocol.labware[labware_id].definitionId
+    except KeyError:
+        raise ProtocolFilesInvalidError(
+            f"Missing loaded labware id: {labware_id}. Invalid protocol."
+        )
```

Comment:
```suggestion
        raise ProtocolFilesInvalidError(
            f"Missing loaded labware id: {labware_id}. Invalid protocol."
        ) from e
```

To change this from:

> We caught a `KeyError` and raised an exception while trying to handle it, which means we failed to handle it.

To:

> We caught a `KeyError` and we're translating it to a `ProtocolFilesInvalidError`.

This doesn't really matter, but it makes the stack trace a little nicer.